scripts/setup-k8s-master.sh.in

```diff
@@ -34,25 +34,39 @@ test -n "{{Region}}"
 test -n "{{ClusterInfoBucket}}"
 test -n "{{ClusterDNSProvider}}"
 
+INSTANCE_ID=$(ec2metadata --instance-id)
```

Comment:
This got moved up because we're using the controlPlaneEndpoint and this needs to be part of that target group before we init, is that right?

Reply:
That's right, all clients connect to the master via the ELB.